.github/workflows/ci-disable-gtest.yml

```diff
@@ -42,21 +42,12 @@ jobs:
     - uses: actions/setup-python@82c7e631bb3cdc910f68e0081d67478d79c6982d # v5.1.0
       with:
         python-version: '3.x'
-    - uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # v1.0.7
-      with:
-        profile: minimal
-        toolchain: stable
-        override: true
+    - name: Set up rust
+      uses: dtolnay/rust-toolchain@stable
```

Comment:
```yml
uses: dtolnay/rust-toolchain@d8352f6b1d2e870bc5716e7a6d9b65c4cc244a1a
```

Reply:
Maybe take the last commit from the `stable` branch so https://github.com/dtolnay/rust-toolchain/commit/d388a4836fcdbde0e50e395dc79a2670ccdef13f

Reply:
Done

Reply:
It is not possible to stick to a commit. Reverting the rust dependency. What really matters in this PR is the cargo install.